Server/Models/Player.cs

```diff
@@ -23,5 +24,27 @@ public class Player
         [Column(TypeName = "jsonb")]
         public Dictionary<string, object>? PatreonCosmetics { get; set; }
         public int Rating12v12 { get; set; }
+        public IEnumerable<PlayerOverthrowRating> PlayerOverthrowRating { get; set; }
```

Comment:
Unless it would complicate usage too much, I'd like to merge these properties. That would make it easier to add more permutations (like solo/party rating) in the future
```suggestion
        public IEnumerable<PlayerRating> Ratings { get; set; }
```